src/store/archive/actions.js

```diff
@@ -0,0 +1,17 @@
+import { bulkRegisterActions } from "../actionHandler"
+import { popOne, pushMany } from "../commonReducers";
+import { matchByEq } from "../utils";
+
+export const actions = bulkRegisterActions({
+  actions: {
+    removeUser: {
+      name: "ARCHIVE_LIST_REMOVE_ONE",
+      effects: { archive: popOne },
+      meta: { matchFn: matchByEq }
```

Comment:
The `popOne` function might be too generic here, it requires the `matchFn` passed as meta-data.  It uses that to do the `findIndex` comparison.  Instead we could probably do `popOneByEquality`, `popOneByPayloadId`, `popOneByItemId` and stuff like that.  We wanna keep the effects readable, however there are cases where we can just have lambdas in there.